superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -61,102 +61,92 @@ const createFetchResourceMethod = (method: string) => (
   return [];
 };
 
-export const getRecentAcitivtyObjs = (
-  userId: string | number,
-  recent: string,
-  addDangerToast: (arg1: string, arg2: any) => any,
-) => {
-  const getParams = (filters?: Array<any>) => {
-    const params = {
-      order_column: 'changed_on_delta_humanized',
-      order_direction: 'desc',
-      page: 0,
-      page_size: 3,
-      filters,
-    };
-    if (!filters) delete params.filters;
-    return rison.encode(params);
+const getParams = (filters?: Array<any>) => {
+  const params = {
+    order_column: 'changed_on_delta_humanized',
+    order_direction: 'desc',
+    page: 0,
+    page_size: 3,
+    filters,
   };
+  if (!filters) delete params.filters;
+  return rison.encode(params);
+};
+
+export const getEditedObjs = (userId: string | number) => {
   const filters = {
-    // chart and dashbaord uses same filters
-    // for edited and created
     edited: [
       {
         col: 'changed_by',
         opr: 'rel_o_m',
         value: `${userId}`,
       },
     ],
-    created: [
-      {
-        col: 'created_by',
-        opr: 'rel_o_m',
-        value: `${userId}`,
-      },
-    ],
   };
-  const baseBatch = [
-    SupersetClient.get({ endpoint: recent }),
+  const batch = [
     SupersetClient.get({
       endpoint: `/api/v1/dashboard/?q=${getParams(filters.edited)}`,
     }),
     SupersetClient.get({
       endpoint: `/api/v1/chart/?q=${getParams(filters.edited)}`,
     }),
-    SupersetClient.get({
-      endpoint: `/api/v1/dashboard/?q=${getParams(filters.created)}`,
-    }),
-    SupersetClient.get({
-      endpoint: `/api/v1/chart/?q=${getParams(filters.created)}`,
-    }),
-    SupersetClient.get({
-      endpoint: `/api/v1/saved_query/?q=${getParams(filters.created)}`,
-    }),
   ];
-  return Promise.all(baseBatch).then(
-    ([
-      recentsRes,
-      editedDash,
-      editedChart,
-      createdByDash,
-      createdByChart,
-      createdByQuery,
-    ]) => {
-      const res: any = {
-        editedDash: editedDash.json?.result.slice(0, 3),
-        editedChart: editedChart.json?.result.slice(0, 3),
-        createdByDash: createdByDash.json?.result.slice(0, 3),
-        createdByChart: createdByChart.json?.result.slice(0, 3),
-        createdByQuery: createdByQuery.json?.result.slice(0, 3),
+  return Promise.all(batch)
```

Comment:
Would there be any perceivable benefit to separating/parallelizing these, too?

Reply:
these two are need to fill out edited so they should load together when both are finished